cli/commands/helpers/print.go

```diff
@@ -58,3 +59,20 @@ func PrintFormatted(flag string, format string, v interface{}, w io.Writer, prin
 		return printToList(v, w)
 	}
 }
+
+// PrintTitle prints a title for tabular format only.
+// Flag overrides the cli config format, if set.
+func PrintTitle(flag string, format string, title string, w io.Writer) error {
+	if flag != "" {
+		format = flag
+	}
+	// checking the formats exclusively to cover invalid formats
+	// that get defaulted to tabular
+	if format != config.FormatJSON && format != config.FormatWrappedJSON && format != config.FormatYAML {
```

Comment:
I'm not sure I understand 100% why this can't be `if format == config.FormatTabular` instead of `!=` all the other types. Given your comment on line 70 I'm assuming I'm missing something!

I think invalid formats get defaulted to `config.DefaultFormat`, which may or may not be equal to `config.FormatTabular`.

Reply:
Invalid formats do get formatted to tabular, but the flag would still represent something else. If we did `if format == config.FormatTabular`, then `--format foo` for example would override the config, and the title would not get printed. See `PrintFormatted` above where the `default` block covers both cases.

Reply:
L66-L67 have the flag override the config format, which is the intended behavior. So it's possible that `format` doesn't yet abide to a defined format.

Reply:
I see, so that `config.DefaultFormat` is only the default value used when the flag is not mentioned at all on the command line. I thought we were rewriting unknown formats to become `config.DefaultFormat` in order to always know all the values that the format variable can take. Maybe we should. Else that branch with all the `!=` will break when/if we add additional formats and forget to update it.